config/test_config.ini

```diff
@@ -3,21 +3,12 @@
 73085629ae85678f90238d45239815ee94237fb2340634355fac24982e245b234 = out:gift:Sending some BTC to Cathy as a gift
 
 # Data loaders
-[dali.plugin.input.csv.manual]
+[dali.plugin.input.csv.manual Main]
 in_csv_file = input/test_manual_in.csv
 out_csv_file =
 intra_csv_file = input/test_manual_intra.csv
 
-[dali.plugin.input.csv.trezor Bob BTC]
```

Comment:
The Trezor wallets can not contain pricing information, so the pair converter plugin is needed when we use them. Removing the trezor wallets and converting them to manual CSVs allows us to add pricing information so that we don't need the pair converters. No pair converters means no need for an internet connection or API call, which means a faster and more performant test.

The holders are still maintained in the CSV files if that is what you are asking about.

Reply:
You're right.